sdk/ai/azure-ai-generative/MANIFEST.in

```diff
@@ -0,0 +1,9 @@
+recursive-include tests *.py
+recursive-include samples *.py *.md *.txt
+include *.md
+include azure/__init__.py
+include azure/ai/__init__.py
+include azure/ai/generative/py.typed
+include azure/ai/generative/operations/component-configs/*
+include azure/ai/generative/index/_utils/encodings/*
```

Comment:
What's the thought on these extra includes with the paths to the code? I understand the one including .txt, but I'm not sure for the others and, if we do include them, why it is not exhaustive

Reply:
there isn't any code in those directories-- those have yaml/json files that need to be included in the package build. Since they only have one type of file, we didn't do the `*.<file extension>` include.